src/components/button/_button.scss

```diff
@@ -175,10 +187,15 @@
       }
     }
 
-    &:hover,
-    &:focus {
+    &:hover {
       background-color: $govuk-secondary-button-hover-colour;
     }
+
+    &:focus {
+      color: $govuk-focus-text-colour;
+      background-color: $govuk-focus-colour;
+      box-shadow: 0 2px 0 $govuk-focus-text-colour;
```

Comment:
It looks like these styles are inherited from .`govuk-button` so maybe we won't need to set them here.

Reply:
The reason there are duplicates is because the secondary buttons :hover state overrides `.govuk-button` focus because its further down the page.